docs/using-the-jdbc-driver/using-plugins/UsingTheReadWriteSplittingPlugin.md

```diff
@@ -11,11 +11,13 @@ final Properties properties = new Properties();
 properties.setProperty(PropertyDefinition.PLUGINS.name, "readWriteSplitting,failover,efm");
 ```
 
-If you would like to use the read-write splitting plugin without the failover plugin, the Aurora host list plugin must be included before the read-write splitting plugin. This informs the driver that it should query for Aurora's topology.
+If you would like to use the read-write splitting plugin without the failover plugin, simply specify `readWriteSplitting` in the plugin chain.
```

Comment:
NIT: If you would like to use the read-write splitting plugin without the failover plugin, make sure you have the `readWriteSplitting` plugin in the plugin property, and that the `failover` plugin is not part of it..